### DISCUSSION TITLE
Cross-Domain Cookie Sharing Issue between WordPress and React.js Servers

### DISCUSSION THREAD

**Discussion by sauravsingh1996** [wp-graphql/wp-graphql#2887] [React]
I'm facing a challenge involving cross-domain cookie sharing between two servers. I have two servers set up as follows:

    WordPress Server:
        URL: http://178.62.80.9
        Purpose: Hosting a WordPress website

    React.js Server:
        URL: http://165.22.113.111
        Purpose: Hosting a React.js application

My goal is to share a cookie with the httponly attribute set to true between these two domains. However, I'm encountering difficulties in achieving this functionality. Despite my efforts, I'm unable to retrieve the cookie on the different domain.

To provide some context, I'm utilizing the following technologies and plugins:

    wp-graphql plugin for WordPress, which provides a GraphQL interface.
    wp-graphql-cors to manage Cross-Origin Resource Sharing (CORS) for the GraphQL endpoint.

I've configured CORS settings to allow communication between these domains, but the issue persists.

My suspicion is that the problem might be related to the nature of cookies and security policies across different domains. I would appreciate any insights or suggestions from the community regarding potential solutions or debugging steps to identify and resolve this issue.

If anyone has experience with cross-domain cookie sharing, especially in the context of WordPress, React.js, and CORS settings, your guidance would be immensely helpful.

**Comment by dotwise** [comment]
What is the main issue? 

If you check in your browser dev tools Application tab, are you seeing that HttpOnly is not being set or that Secure is not being set? 

In my experience both need to be true to be able to share login between WordPress and React.
